arch/migrations/0013_auto_20210603_2343.py

```diff
@@ -145,12 +145,12 @@
     "PUMaC Finals": "PUF",
 }
 for k, v in list(lookup.items()):
-    lookup[k + " MO"] = v + "MO"
-    lookup[k + " TST"] = v + "TST"
-    lookup[k + " RMM TST"] = v + "RST"
-    lookup[k + " EGMO TST"] = v + "EST"
-    lookup[k + " JBMO TST"] = v + "JST"
-    lookup[k + " TSTST"] = v + "TSTST"
+    lookup[f"{k} MO"] = f"{v}MO"
+    lookup[f"{k} TST"] = f"{v}TST"
+    lookup[f"{k} RMM TST"] = f"{v}RST"
+    lookup[f"{k} EGMO TST"] = f"{v}EST"
+    lookup[f"{k} JBMO TST"] = f"{v}JST"
+    lookup[f"{k} TSTST"] = f"{v}TSTST"
```

Comment:
refactoring a migration seems a bit scary

Reply:
i think it's equivalent, so it should be fine... i hope...